robohive/envs/env_base.py

```diff
@@ -130,7 +133,7 @@ def _setup(self,
         self._setup_rgb_encoders(self.visual_keys, device=None)
 
         # reset to get the env ready
-        observation, _reward, done, _info = self.step(np.zeros(self.sim.model.nu))
+        observation, _reward, done, *_, _info = self.step(np.zeros(self.sim.model.nu))
```

Comment:
We could decorate this too but it's also ok if you don't need the truncated oc

Reply:
Seems to work for now. Lets revisit in the future incase this isn't sufficient